indigo_drivers/mount_ioptron/README.md

```diff
@@ -20,11 +20,15 @@ indigo_server indigo_mount_ioptron
 
 ## Comments
 
-Use URL in form ieq://host:port to connect to the mount over network (default port is 4030).
+Use URL in form ieq://host:port to connect to the mount over network. The default port can be found through the hand controller going to Settings < Wi-Fi Options < Wireless Status
+
+**EXAMPLE OF SETTING IN INDIGO**
+
+![Alt text](ioptronsettings.jpg)
```

Comment:
I think screenshots do not add any value as different clients show it differently and they do not show the full path to the property. It is better to descibe it in a single sentence.